plugins/main/public/components/common/wazuh-discover/render-columns.tsx

```diff
@@ -3,26 +3,23 @@ import { EuiLink } from '@elastic/eui';
 import { tDataGridRenderColumn } from '../data-grid';
 import { getCore } from '../../../kibana-services';
 import { RedirectAppLinks } from '../../../../../../src/plugins/opensearch_dashboards_react/public';
-import { endpointSummary, rules } from '../../../utils/applications';
-import { formatUIDate } from '../../../react-services';
+import {
+  endpointSummary,
+  rules,
+  mitreAttack,
+} from '../../../utils/applications';
 import NavigationService from '../../../react-services/navigation-service';
 
 export const MAX_ENTRIES_PER_QUERY = 10000;
 
-const navigateTo = (ev, section, params) => {
-  NavigationService.getInstance().navigateToModule(ev, section, params);
-};
-
-const renderMitreTechnique = (technique: string) => (
+const renderMitreTechnique = technique => (
   <EuiLink
-    onClick={e =>
-      navigateTo(e, 'overview', {
-        tab: 'mitre',
-        tabView: 'intelligence',
-        tabRedirect: 'techniques',
-        idToRedirect: technique,
-      })
-    }
+    onClick={e => {
+      e.preventDefault();
+      NavigationService.getInstance().navigateToApp(mitreAttack.id, {
+        path: `#/overview?tab=mitre&tabView=intelligence&tabRedirect=techniques&idToRedirect=${technique}`,
+      });
+    }}
   >
     {technique}
   </EuiLink>
```

Comment:
Perhaps you could use the RedirectAppLinks to get the reference to where you are navigating to. As it is in line 35.

```
  <RedirectAppLinks application={getCore().application}>
    <EuiLink
      href={NavigationService.getInstance().getUrlForApp(mitreAttack.id, {
        path: `#/overview?tab=mitre&tabView=intelligence&tabRedirect=techniques&idToRedirect=${technique}`,
      })}
    >
      {technique}
    </EuiLink>
  </RedirectAppLinks>
```

<img width="1287" alt="image" src="https://github.com/user-attachments/assets/c365b866-10b1-410c-90e5-4e2ddff78334">

Reply:
Fix in commit https://github.com/wazuh/wazuh-dashboard-plugins/pull/6845#discussion_r1684130631